pkg/read/read.go

```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 )
 
-const eol = "\n"
+const Eol = "\n"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported const Eol should have comment or be unexported](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498955)